pyfolio/tests/test_timeseries.py

```diff
@@ -311,12 +311,13 @@ class TestStats(TestCase):
         (simple_month_rets, utils.MONTHLY, 0.006885932704891129)
     ])
     def test_annual_ret(self, returns, period, expected):
-        self.assertEqual(
+        assert_almost_equal(
```

Comment:
In `cum_returns` there was a slight change in the calculation. `np.log(1 + returns)` was changed to `np.log1p(returns)` and that is causing the slight difference in the values for 'cum_returns`, `annual_return`, and `aggregate_returns`.